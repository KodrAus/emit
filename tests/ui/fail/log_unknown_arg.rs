#![feature(stmt_expr_attributes, proc_macro_hygiene)]

fn main() {
    tracing_subscriber::fmt().init();

    let v = "Some data";

    emit::info!(not_an_arg: false, "Logging with a deeply nested {value: v}");
}
