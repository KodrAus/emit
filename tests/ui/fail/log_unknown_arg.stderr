error: unexpected field `not_an_arg`
 --> fail/log_unknown_arg.rs:8:17
  |
8 |     emit::info!(not_an_arg: false, "Logging with a deeply nested {value: v}");
  |                 ^^^^^^^^^^^^^^^^^
